packages/cloudapi-gql/src/api/policies.js

```diff
@@ -1,21 +1,23 @@
 const request = require('./request');
+// const aperture = require('aperture');
```

Comment:
Is this going to be added later ?

Reply:
yes, possibly. refers to this: https://github.com/joyent/node-aperture